pkgs/top-level/all-packages.nix

```diff
@@ -15009,7 +15009,7 @@ with pkgs;
   inherit (gnome3) evince;
   evolution_data_server = gnome3.evolution_data_server;
 
-  keepass = callPackage ../applications/misc/keepass { 
+  keepass = callPackage ../applications/misc/keepass {
```

Comment:
I'm all for removing trailing whitespace, but probably best to do so separately.

Reply:
emacs trims it automatically but I could split it out of course.

Reply:
Your call, folks don't seem to be particular about this.